patches/minecraft/net/minecraft/enchantment/EnchantmentHelper.java.patch

```diff
@@ -1,6 +1,19 @@
 --- ../src-base/minecraft/net/minecraft/enchantment/EnchantmentHelper.java
 +++ ../src-work/minecraft/net/minecraft/enchantment/EnchantmentHelper.java
-@@ -292,7 +292,7 @@
+@@ -30,6 +30,12 @@
+     private static final EnchantmentHelper.HurtIterator field_151388_d = new EnchantmentHelper.HurtIterator();
+     private static final EnchantmentHelper.DamageIterator field_151389_e = new EnchantmentHelper.DamageIterator();
+ 
++    public static int getEnchantmentLevel(EntityLivingBase entityLivingBase, Enchantment enchID, ItemStack stack)
++    {
```

Comment:
It's customanry to put the new parameters at the END of the function so that it makes it easier to associate the new one. Plus, having the parameters line up in each method is better style.